modules/ballerina-http/src/main/ballerina/ballerina/net/ws/annotation.bal

```diff
@@ -1,11 +1,11 @@
 package ballerina.net.ws;
 
 @Description {value: "Configuration for a WebSocket service."}
-@Field {value: "basePath: Path of the WebSocket service."}
-@Field {value: "subProtocols: Negotiable sub protocol by the service."}
-@Field {value:"host: Host of the service."}
-@Field {value:"port: Port number of the service."}
-@Field {value:"wssPort: WSS port number of service."}
+@Field {value: "basePath: Path of the WebSocket service"}
```

Comment:
Shall we remove the extra space after "value:"

Reply:
Ack.